### ISSUE TITLE
do not block psn if evilnat block is active

### ISSUE THREAD

**Issue by hardlevel** [aldostools/webMAN-MOD#829]
Hi! 

Any chances to add a function to do not block psn access (that screen tha ask to press R2+ Triangle) if the evilnat option to do the same is active?

By default if we use the last evilnat and the last webman, we have two options to do the same, and I had a problem, I made the unlock on evilnat, but the block of webman mod still there, and I could not unblock it until I disabled the security option on evilnat

in other words, there's two options to do the same thing and both are conflitcting somehow

thanks!

**Comment by aldostools** [comment]
Evilnat 4.89.3 has an option in CFW Tools menu that let disable the PSN Blocker to avoid conflicts.

**Comment by hardlevel** [comment]
yes... i know, but if I prefer to use the option of evilnat instead of the
webman mod?

one more thing, any chances of webman mod do a dump of discs?

**Comment by aldostools** [comment]
Then we would have redundant features again.

Although you can copy /dev_bdvd to a folder in /dev_hdd0 is background then convert the folder to ISO with a backup manager, some users have reported issues.

To dump discs I recommend to use multiMAN or IRISMAN (you should enable BDVD drive in START menu)
